Assignment_05.rb

```diff
@@ -61,7 +61,7 @@ def occupation_percecntage_day_wise(total_time_in_minute)
 
   # This method does the validation
   def valid?
-    validate_time_logs
+    validate_input_format && validate_time_logs
```

Comment:
@AritraChatterjee-qi Can you please rename and segregate the validation parts based on the input fields -> eg: 
1. validates_id
2. validates_name
3. validates_time_logs

Reply:
Can you please remove the default id  and name mentioned in initialize method?